JS/Lesson24/tugba/map.js

```diff
@@ -178,12 +181,20 @@ const studentsExercise5 = [
 		grade: 'B-',
 	},
 ];
+const studentNamesAndGrades = studentsExercise5.map(student => `${student.name} ${student.grade}`);
+
+console.log(studentNamesAndGrades);
 
 
 // Exercise 6:
 // Given an array of numbers, convert each number to a string representation using .map().
 const numbersExercise6 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+const stringRepresentation = numbersExercise6.map(number => String(number));
```

Comment:
In order to convert something to a string in JS, you do like this: `variableNameToConvert.toString();`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString

```suggestion
const stringRepresentation = numbersExercise6.map(number =>number.toString());
```